Tiltfile

```diff
@@ -0,0 +1,66 @@
+# This loads a helper function that isn't part of core Tilt that simplifies restarting the process in the container
+# when files changes.
+load('ext://restart_process', 'docker_build_with_restart')
+
+# Treat the main binary as a local resource, so we can automatically rebuild it when any of the deps change. This
+# builds it locally, targeting linux, so it can run in a linux container.
+local_resource(
+    'manager_binary',
+    cmd="""
+mkdir -p .tiltbuild/bin
+CGO_ENABLED=0 GOOS=linux go build -o .tiltbuild/bin/manager ./cmd/manager
+""",
```

Comment:
Double-quotes are used here, single-quotes are used below. Does it matter? Is consistency important?

Reply:
Doesn't matter